pkg/plugins/policies/meshratelimit/plugin/xds/configurer.go

```diff
@@ -58,6 +58,7 @@ func RateLimitConfigurationFromPolicy(rl *api.LocalHTTP) *envoy_routes_v3.RateLi
 type Configurer struct {
 	Element subsetutils.Element
 	Rules   core_rules.Rules
+	Conf    *api.Conf
```

Comment:
You can remove both `Element` and `Rules` and put the required `Conf` right in the `plugin.go`

Reply:
this is not that simple, as this is needed for gateway configuration: https://github.com/kumahq/kuma/blob/820494cf59e22d5a4870ca6d7a01f1f77e38ad46/pkg/plugins/policies/meshratelimit/plugin/xds/configurer.go#L129 as we have specific config per route

Reply:
I see 😞